### DISCUSSION TITLE
Apple privacy manifest file

### DISCUSSION THREAD

**Discussion by yamade** [software-mansion/react-native-gesture-handler#2751] [Q&A] [answered]
Hello

You guys may notice this announcement from Apple.
https://developer.apple.com/support/third-party-SDK-requirements/

I think react-native-gesture-handler is using `mach_absolute_time`.
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api#4278394

So I'm wondering if you guys have any plan to add a privacy manifest file sometime soon. Thank you!

**Comment by j-piasecki** [comment, marked as answer]
Yes, it will be present in the next release (https://github.com/software-mansion/react-native-gesture-handler/pull/2820) and most likely backported to the most recent minors.

**Reply by yamade** [reply]
Thanks!
